### DISCUSSION TITLE
Option to show stream previews on tabs

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5132] [Ideas]
Since tabs can be at the bottom, you have to fly through the entire window to see a preview of the stream. This is inconvenient, so I suggest adding an option to choose where to show stream previews: on the chat title bar or on the tabs.

**Comment by x9136** [comment]
Closed due no response.

**Reply by Nerixyz** [reply]
I'm sorry, but you can't expect people, especially in open source, to implement all features in a month. If a discussion in the category "Ideas" didn't receive any comments, then it's a decent idea (it hasn't been closed!), but no one has it at the top of their priority list.

**Reply by x9136** [reply]
The idea has not been deleted. If someone is interested, you can always find the idea in the search and reopen it.

I close ideas that don't get attention for a significant amount of time or aren't planned for development for a long time, because that means they're not interesting or at least in the very last priority relative to thousands of other tasks.

**Reply by Felanbird** [reply]
> because that means they're not interesting or at least in the very last priority relative to thousands of other tasks.

Well you're wrong, #5119 is for an issue opened in November 2020
#5186 is for an issue opened in September 2020
#5176 was for an issue opened in April 2021
#5118 was for an issue opened in Feb 2021
#5111 was for an issue opened in March 2021
#5063 was for an issue opened in July 2021

**Reply by x9136** [reply]
@Felanbird You say that as if it mattered to the development whether that ideas were closed or not.

If you are interested in an idea that I close, you have everything you need to reopen it. I am not going to close any ideas twice.

**Reply by Felanbird** [reply]
I don't personally care about your closing of issues, none of your ideas have been interesting to me, which I why I never prompted you about why you were doing this. That being said, if you actually want your ideas added, you are going about it the wrong way :+1:
